cpe_crypt/resources/cpe_crypt_authdb.rb

```diff
@@ -25,7 +25,7 @@
   # First, validate whether or not the current settings are correct
   # Get current settings from authdb
   if crypt_currently_in_authdb && !uninstall?
-    log('Authdb already configured for Crypt')
+    Chef::Log.info('Authdb already configured for Crypt')
```

Comment:
Why did you make this change? log will be run time whereas Chef::Log.info() will be compile time. I think its more clear to use the log resource.

Reply:
While I agree about the run time vs compile time statement this message is really only useful for troubleshooting. Also, `log` shows up as an updated resource when chef is ran via `chef-client`. This causes reports to show false updates.

Reply:
Oh what chef version are you on? on my version it doesn't.
But in that case should the level be debug ?

Reply:
I just replicated on chef versions 12.21.4, 13.4.19, & 13.5.3. All of them showed `log` as an updated resource. As for being info vs debug message I agree and pushed an updated commit.